src/applications/personalization/dashboard/reducers/folders.js

```diff
@@ -64,6 +68,34 @@ export default function folders(state = initialState, action) {
       );
     }
 
+    case FETCH_INBOX_SUCCESS: {
```

Comment:
This reducer was mostly cut and paste from the existing FETCH_FOLDER_SUCCESS reducer above.